football/spec/models/user_spec.rb

```diff
@@ -2,15 +2,32 @@
 
 RSpec.describe :User, type: :model do
 
-  describe "validate email" do 
-    before(:each) do
+  before(:all) do
       @user = FactoryGirl.build(:user)
     end
-    subject { @user }
+  subject { @user }
+
+  describe "validate email" do 
 
     it { @user.should_not be_nil }
   	it { should validate_presence_of :email}
 
   end
 
+  describe "validate smartphone" do 
```

Comment:
Sometimes we don't need `describe "blah blah" do` when we write rspec for validations